README.md

```diff
@@ -244,9 +343,19 @@ You can also delete directories using `rm` or `rmdir`.
 
```

Comment:
This section needs more spelling out: recursive deletion is a unix is a chainsaw moment.

Also fair to show that `rm -rf /` is a Bad Idea (tm)